.github/actions/setup-build/action.yml

```diff
@@ -15,6 +15,7 @@ runs:
 
   steps:
   - name: Set up Python
+    if: ${{ runner.arch == 'X64' }}
```

Comment:
Should we add a similar guard condition around the `pip install` lines below as well?

Also, let's add a comment to explain why this line is necessary/sufficient.

Reply:
Oh, I see.  Instead of setting up Python explicitly, are we using the Python version installed on the system?

Reply:
Yes. Because GitHub doesn't provide arm64 python